modules/model/item/disease.js

```diff
@@ -1,59 +1,67 @@
-import { BaseItemModel } from "./components/base";
+import {BaseItemModel} from "./components/base";
+
 let fields = foundry.data.fields;
 
 /**
  * Represents an Item used by both Patrons and Characters/NPCs
  */
-export class DiseaseModel extends BaseItemModel
-{
-
-    static defineSchema() 
-    {
-        let schema = {};
-        schema.contraction = new fields.SchemaField({
-            value : new fields.StringField(),
-        });
-        
-        schema.incubation = new fields.SchemaField({
-            value : new fields.StringField(),
-        })
-
-        schema.duration = new fields.SchemaField({
-            value : new fields.StringField(),
-            unit : new fields.StringField(),
-            active : new fields.BooleanField(),
-        })
-
-        schema.symptoms = new fields.SchemaField({
-            value : new fields.StringField(),
-        })
-
-        schema.permanent = new fields.SchemaField({
-            value : new fields.StringField(),
-        })
-        return schema;
-    }
-
-    async expandData(htmlOptions) {
-        let data = await super.expandData(htmlOptions);
-        data.properties.push(`<b>${game.i18n.localize("Contraction")}:</b> ${this.contraction.value}`);
-        data.properties.push(`<b>${game.i18n.localize("Incubation")}:</b> ${this.incubation.value} ${this.incubation.unit}`);
-        data.properties.push(`<b>${game.i18n.localize("Duration")}:</b> ${this.duration.value} ${this.duration.unit}`);
-        data.properties = data.properties.concat(this.parent.effects.map(i => i = "<a class ='symptom-tag'><i class='fas fa-user-injured'></i> " + i.name.trim() + "</a>").join(", "));
-        if (this.permanent.value)
-          data.properties.push(`<b>${game.i18n.localize("Permanent")}:</b> ${this.permanent.value}`);
-        return data;
-      }
-
-      chatData()
-      {
-        let properties = [];
-        properties.push(`<b>${game.i18n.localize("Contraction")}:</b> ${this.contraction.value}`);
-        properties.push(`<b>${game.i18n.localize("Incubation")}:</b> <a class = 'chat-roll'><i class='fas fa-dice'></i> ${this.incubation.value}</a>`);
-        properties.push(`<b>${game.i18n.localize("Duration")}:</b> <a class = 'chat-roll'><i class='fas fa-dice'></i> ${this.duration.value}</a>`);
-        properties.push(`<b>${game.i18n.localize("Symptoms")}:</b> ${(this.symptoms.value.split(",").map(i => i = "<a class ='symptom-tag'><i class='fas fa-user-injured'></i> " + i.trim() + "</a>")).join(", ")}`);
-        if (this.permanent.value)
-          properties.push(`<b>${game.i18n.localize("Permanent")}:</b> ${this.permanent.value}`);
-        return properties;
-      }
```

Comment:
Whole file shows as changed becuase of formatting, actual change can be seen in specific commit: https://github.com/moo-man/WFRP4e-FoundryVTT/pull/1851/commits/ed6fe9651f0bf56b074e4b23d1f8db0e2e7ceee7